src/Nethermind/Nethermind.Runner/Nethermind.Runner.csproj

```diff
@@ -101,6 +101,9 @@
     <None Remove="out\**" />
     <None Remove="keystore\**" />
   </ItemGroup>
+  <ItemGroup>
+    <None Remove="configs\eip4844_docker.cfg" />
```

Comment:
Cosmetic: to remove?

Reply:
Yes, let me remove it. It will be needed to participate in the devnet, but let it be just there, outside of our repo